phivk/gradu.al
# Feature/password page protect

This adds a simple password protect to the COEE community.

To test:
- Add a `COEE_PASSWORD` field to your .env file
- Restart Nuxt
- Navigate to /coee
  - You should be redirected to /coee/password and see a password field
- Use the password you created in the .env file
- You should be redirected back to the COEE page.
